src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/ViaPlanResponse.java

```diff
@@ -0,0 +1,22 @@
+package org.opentripplanner.ext.transmodelapi.model.plan;
+
+import java.util.List;
+import org.opentripplanner.model.plan.Itinerary;
+import org.opentripplanner.routing.api.response.RoutingError;
+import org.opentripplanner.routing.api.response.ViaRoutingResponse;
+
+public record ViaPlanResponse(
+  List<List<Itinerary>> viaJourneys,
+  List<List<List<Integer>>> viaJourneyConnections,
+  List<RoutingError> routingErrors
+) {
```

Comment:
This need a proper model like the one we did for schema, then JavaDoc.

Reply:
I removed this class, as it no longer serves any purpose, and just use `ViaRoutingResponse` in the Transmodel API